docs/carbon-as-spark-datasource-guide.md

```diff
@@ -44,22 +45,20 @@ Carbon table can be created with spark's datasource DDL syntax as follows.
 | table_blocksize | 1024 | Size of blocks to write onto hdfs |
 | table_blocklet_size | 64 | Size of blocklet to write |
 | local_dictionary_threshold | 10000 | Cardinality upto which the local dictionary can be generated  |
-| local_dictionary_enable | false | Enable local dictionary generation  |
-| sort_columns | all dimensions are sorted | comma separated string columns which to include in sort and its order of sort |
-| sort_scope | local_sort | Sort scope of the load.Options include no sort, local sort ,batch sort and global sort |
-| long_string_columns | null | comma separated string columns which are more than 32k length |
+| local_dictionary_enable | false | Enable local dictionary generation |
+| sort_columns | all dimensions are sorted | Comma separated string columns which to include in sort and its order of sort |
```

Comment:
int columns can be included in sort_columns.Remove the word string.
Change the grammar of the sentence.